docs/usermanual/userman_admin.rst

```diff
@@ -446,9 +446,8 @@ After registering your device:
 
 .. screenshot::
     :user-role: admin
-    :url: /management/zone/
+    :url: /update/videos/
     :navigation-steps:
-    :focus: li.video
     :class: screenshot
```

Comment:
Could you test adding `:alt: alternate text` to one screenshot before you merge, make sure it works, and I'll add the rest when I commit my changes to docs?